scalding-core/src/main/scala/com/twitter/scalding/serialization/CascadingBinaryComparator.scala

```diff
@@ -53,6 +54,7 @@ object CascadingBinaryComparator {
 
     def check(s: Splice): Try[Unit] = {
       val m = s.getKeySelectors.asScala
+      val sortingSelectors = s.getSortingSelectors.asScala
```

Comment:
can't we just do:

```scala
val m = s.getKeySelectors.asScala ++ s.getSortingSelectors.asScala
```
since the check is exactly the same? then we don't have to duplicate the checking code.